samwise.js

```diff
@@ -4,20 +4,20 @@
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?